metashare/consumers.py

```diff
@@ -32,17 +33,47 @@ class PushNotificationConsumer(AsyncJsonWebsocketConsumer):
 
             channel_layer.group_send(group_name, {
                 'type': 'notify',  # This routes it to this handler.
-                'content': json_message,
+                'content': {
+                    'type': str (will map to frontend Redux event),
+                    'payload': {
+                        'originating_user_id': str,
+                        'message': str (error message, optional),
+                    },
+                    'model_name': str,
+                    'id': str,
+                },
             })
         """
         # Take lock out of redis for this message:
         await clear_message_semaphore(self.channel_layer, event)
         if "content" in event:
-            await self.send_json(event["content"])
+            message = await self.hydrate_message(event["content"])
+            await self.send_json(message)
             return
 
+    async def hydrate_message(self, content):
+        content = deepcopy(content)
+        model_name = content.pop("model_name")
+        id = content.pop("id")
+        # We specifically don't want to include every user, as that
+        # would cause every generic-message to include the user who's
+        # getting the message to be included. It'd just be noise on the
+        # wire.
+        if model_name.lower() != "user":
+            try:
+                instance = await self.get_instance(model=model_name, id=id)
+            except ObjectDoesNotExist:
+                pass
```

Comment:
@wlonk Hm, this won't quite work. We don't need the full serialized model, but on the front end we still need to know the Org `task`, `org_type`, and `id`.